packages/salesforcedx-vscode-apex/package.json

```diff
@@ -53,7 +53,8 @@
     "test": "node ../../scripts/run-test-with-top-level-extensions",
```

Comment:
Is there a difference between the `test` specified here and the new `test:vscode-integration` category? Or are they both running the same script?

Reply:
I think that it would be nice to keep a script that runs all of the tests (regardless of category) within a package. I could change the `test` script to call all the scripts for the test sub-categories, but I ran into problems with code coverage when I tried that. Let me look into it some more.